cratedb_toolkit/cluster/core.py

```
@@ -619,6 +633,31 @@ def load_table(
 
         return self
 
+    def save_table(
+        self, source: TableAddress, target: InputOutputResource, transformation: t.Union[Path, None] = None
```

Ruff (0.15.1)

[warning] Unused method argument: `source`

(ARG002)